hax/treemakers/posrec.py

```diff
@@ -23,7 +23,9 @@ class PositionReconstruction(TreeMaker):
        - s1_pattern_fit_bottom_hits_hax: S1 pattern likelihood computed with corrected
                                          position and bottom array hits
 
-       - s1_area_fraction_top_probability_hax: S1 AFT computed with corrected position
+       - s1_area_fraction_top_probability_hax: S1 AFT p-value computed with corrected position
+       - s1_area_fraction_top_probability_nothresh: computed using PE below 10 (instead of hits)
+       - s1_area_fraction_top_map_value: Expected S1 AFT from map
```

Comment:
I had intended that we store the value of the binomial pmf (`binom_pmf` from the plugin). Expected S1 AFT isn't hard to figure out (varies pretty closely with z), but the pmf is probably more useful in edge cases.

Reply:
oh ok. how about now?